openpgp/keys_test.go

```diff
@@ -1155,12 +1303,18 @@ func TestRevokeSubkeyWithConfig(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	if sk.Sig.Hash != c.DefaultHash {
+	if len(sk.Revocations) != 1 {
+		t.Fatalf("Expected 1 subkey revocation signature, got %v", len(sk.Revocations))
+	}
+
+	revSig := sk.Revocations[0]
+
+	if revSig.Hash != c.DefaultHash {
 		t.Fatalf("Expected signature hash method: %v, got: %v", c.DefaultHash,
-			sk.Sig.Hash)
+		revSig.Hash)
```

Comment:
Why a newline?

Reply:
This is the autoformatter doing stuff, I'll change it, thanks for spotting :+1: